robottelo/utils/virtwho.py

```diff
@@ -473,6 +473,33 @@ def hypervisor_json_create(hypervisors, guests):
     return mapping
 
 
+def hypervisor_fake_json_create(hypervisors, guests):
+    """
+    Create a hypervisor guest json data for fake config usages. For example:
+    {'hypervisors': [{'uuid': '820b5143-3885-4dba-9358-4ce8c30d934e',
+    'guests': [{'guestId': 'afb91b1f-8438-46f5-bc67-d7ab328ef782', 'state': 1,
+    'attributes': {'active': 1, 'virtWhoType': 'esx'}}]}]}
+    :param hypervisors: how many hypervisors will be created
+    :param guests: how many guests will be created
+    """
+    hypervisors_list = []
+    for _ in range(hypervisors):
+        guest_list = []
+        for _ in range(guests):
+            guest_list.append(
+                {
+                    "guestId": str(uuid.uuid4()),
+                    "state": 1,
+                    "attributes": {"active": 1, "virtWhoType": "esx"},
```

Comment:
> parametrized?

@lhellebr Could you please describe it in detail? thank you

Reply:
In other words, should this test be universal for different `virtWhoType`s, rather than `esx` only? For example, `libvirt`? You would use `@parametrize` to make this test test both.

Reply:
Hello @lhellebr  For some cases we only run on esx, and esx env cover all the functions, the other hypervisors will test the basic and specific functions only belong this hypervisor, and this case we chose only run on esx.

Reply:
I think the BZ should be tested with all hypervisors though.
You're the SME, I'll leave this bit up to you.